### PR TITLE
primary_transcript as a supported feature and remove TIGRFAM xrefs

### PR DESCRIPTION
**Pull request by Dishalodha**
1. Added support for primary_transcript
2. TIGRFAM has been removed from xref analysis and interproscan

### CODE REVIEW

File: config/external_db_map/default.txt [hunk 1]
```diff
@@ -209,6 +209,7 @@ MARRVEL_MODEL	_IGNORE_
 NRL_3D	_IGNORE_
 SignaLink	_IGNORE_
 TF	_IGNORE_
+TIGRFAM	_IGNORE_
```
Comment (vsitnik) [inline config/external_db_map/default.txt:212]: could you add an empty line? Thanks

Comment (Dishalodha) [reply config/external_db_map/default.txt:212]: ah yes we do the same here eg: however we keep it as `primary_transcript` 
```python
###
NC_072473.1     BestRefSeq	gene    5432717 5432841 .	+	.	ID=GeneID_104797198
NC_072473.1     BestRefSeq	primary_transcript	5432717 5432841 .	+	.	ID=GeneID_104797198_t1;Parent=GeneID_104797198
NC_072473.1     BestRefSeq	exon    5432717 5432841 .	+	.	ID=GeneID_104797198_t1-E1;Parent=GeneID_104797198_t1
###
NC_072473.1     BestRefSeq	gene    5432747 5432771 .	+	.	ID=TMP_efusBU_THK_EF1_1
NC_072473.1     BestRefSeq	miRNA   5432747 5432771 .	+	.	ID=TMP_efusBU_THK_EF1_1_t1;Parent=TMP_efusBU_THK_EF1_1
NC_072473.1     BestRefSeq	exon    5432747 5432771 .	+	.	ID=TMP_efusBU_THK_EF1_1_t1-E1;Parent=TMP_efusBU_THK_EF1_1_t1

```

Comment (vsitnik) [reply config/external_db_map/default.txt:213]: we have pre_mirna:
https://github.com/Ensembl/ensembl-genomio/pull/351/files#diff-9809d474df8cf1bea4f704049d61b2095437726bd23661b2842de8ac329bed66R83

up to you,  though

**Review by vsitnik — APPROVED** [review]
ok

**Pull request merged**